doc/source/missing_data.rst

```diff
@@ -176,7 +176,16 @@ NA values in GroupBy
 NA groups in GroupBy are automatically excluded. This behavior is consistent
 with R, for example.
 
+<<<<<<< HEAD
 See the groupby section :ref:`here <groupby.missing>` for more information.
+=======
+.. ipython:: python
```

Comment:
you need to remove the conflict lines
further I would move your change above the see further reference